src/components/Exchange/FeesTooltip.tsx

```diff
@@ -0,0 +1,122 @@
+import { i18n, MessageDescriptor } from "@lingui/core";
+import { defineMessage, Trans } from "@lingui/macro";
+import ExternalLink from "components/ExternalLink/ExternalLink";
+import StatsTooltipRow from "components/StatsTooltip/StatsTooltipRow";
+import Tooltip from "components/Tooltip/Tooltip";
+import { getConstant } from "config/chains";
+import { BigNumber, BigNumberish } from "ethers";
+import { useChainId } from "lib/chains";
+import { USD_DECIMALS } from "lib/legacy";
+import { formatAmount, formatAmountFree } from "lib/numbers";
+
+type Fee = { label: string; value: string };
+type ExecutionFee = { fee?: BigNumberish; feeUSD?: BigNumberish };
+type FeeType = "open" | "close" | "swap" | "borrow" | "deposit" | "execution";
+
+const FEE_LABELS: Record<FeeType, MessageDescriptor> = {
+  open: defineMessage({ message: "Open Fee" }),
+  close: defineMessage({ message: "Close Fee" }),
+  swap: defineMessage({ message: "Swap Fee" }),
+  borrow: defineMessage({ message: "Borrow Fee" }),
+  deposit: defineMessage({ message: "Deposit Fee" }),
+  execution: defineMessage({ message: "Execution Fee" }),
+};
+
+function getExecutionFeeStr(chainId, executionFee, executionFeeUsd) {
+  const nativeTokenSymbol = getConstant(chainId, "nativeTokenSymbol");
+
+  if (!nativeTokenSymbol || !executionFee || !executionFeeUsd) {
+    return "";
+  }
+
+  const formattedExecutionFee = formatAmountFree(executionFee, 18, 4);
+  const formattedExecutionFeeUsd = formatAmount(executionFeeUsd, USD_DECIMALS, 2);
+  return `${formattedExecutionFee} ${nativeTokenSymbol} ($${formattedExecutionFeeUsd})`;
+}
+
+function getFeesInStr(fees: BigNumber | undefined): string {
```

Comment:
maybe just getFeesStr? as in getExecutionFeeStr

Reply:
Fixed and updated!